chromium_src/chrome/browser/importer/importer_list.cc

```diff
@@ -46,6 +46,7 @@ void AddChromeToProfiles(std::vector<importer::SourceProfile>* profiles,
     chrome.importer_name = base::UTF8ToUTF16(base::StrCat({brand, " ", *name}));
     chrome.importer_type = type;
     chrome.services_supported = items;
+    chrome.profile = base::UTF8ToUTF16(brand);
```

Comment:
Is this sufficient for multiple profiles from the same browser?

Reply:
yes, this parameter was not used, we are putting multiple profiles to the profiles list bellow and later in webui we match the browser to this branch, profiles for import selected from an array by index